korman/ui/modifiers/render.py

```diff
@@ -196,6 +196,8 @@ def lightmap(modifier, layout, context):
     col = layout.column()
     col.active = is_texture
     col.prop_search(modifier, "uv_map", context.active_object.data, "uv_textures")
+    if bool(modifier.id_data.modifiers):
```

Comment:
```suggestion
    if bool(modifier.id_data.modifiers and modifier.uv_map):
```
No need to display the warning if the user specifies no UVmap, as we'll generate it ourselves.

Reply:
This suggestion won't work as written, blender sequences do not implicitly convert to bool (bug in blender). So it needs to be:
```suggestion
    if bool(modifier.id_data.modifiers) and modifier.uv_map:
```

Reply:
Oops, right. Probably a copy paste error.